packages/squiggle-lang/__tests__/Distributions/KlDivergence_test.res

```diff
@@ -96,6 +116,62 @@ describe("klDivergence: discrete -> discrete -> float", () => {
   })
 })
 
+describe("klDivergence: mixed -> mixed -> float", () => {
+  let klDivergence = DistributionOperation.Constructors.klDivergence(~env)
+  let mixture = a => DistributionTypes.DistributionOperation.Mixture(a)
+  let a' = [(point1, 1.0), (uniformDist, 1.0)]->mixture->run
+  let b' = [(point1, 1.0), (floatDist, 1.0), (normalDist10, 1.0)]->mixture->run
+  let c' = [(point1, 1.0), (point2, 1.0), (point3, 1.0), (uniformDist, 1.0)]->mixture->run
+  let d' =
+    [(point1, 1.0), (point2, 1.0), (point3, 1.0), (floatDist, 1.0), (uniformDist2, 1.0)]
+    ->mixture
+    ->run
+  let (a, b, c, d) = switch (a', b', c', d') {
+  | (Dist(a''), Dist(b''), Dist(c''), Dist(d'')) => (a'', b'', c'', d'')
+  | _ => raise(MixtureFailed)
+  }
```

Comment:
lines 120-132 should be turned into a function or something. Really though, I'd expect it to be two functions (1 for a and b, 1 for c and d) or something.